### ISSUE TITLE
Low accuracy when using rmsprop

### ISSUE THREAD

**Issue by denru01** [marvis/pytorch-mobilenet#9]
I encountered the same issue that the accuracy given by rmsprop is low. I also tried other optimizers and the results are bad too. The only working optimizer is SGD. I did not encounter this issue with Tensorflow or other toolboxes. Do you have any idea why Pytorch has this problem?

Thanks!
